public/consolidated/all_snippets.json

```diff
@@ -371,6 +371,99 @@
           "utility"
         ],
         "author": "dostonnabotov"
+      },
+      {
+        "title": "Truncate Text",
+        "description": "Truncates the text to a maximum length and appends '...' if the text exceeds the maximum length.",
+        "code": [
+          "const truncateText = (text = '', maxLength = 50) => {",
+          "  return `${text.slice(0, maxLength)}${text.length >= maxLength ? '...' : ''}`;",
+          "};",
+          "",
+          "// Usage:",
+          "const title = \"Hello, World! This is a Test.\";",
+          "console.log(truncateText(title)); // Output: 'Hello, World! This is a Test.'",
+          "console.log(truncateText(title, 10)); // Output: 'Hello, Wor...'"
+        ],
+        "tags": ["javascript", "string", "truncate", "utility", "text"],
+        "author": "realvishalrana"
+      },
+      {
+        "title": "Data with Prefix",
+        "description": "Adds a prefix and postfix to data, with a fallback value.",
+        "code": [
+          "const dataWithPrefix = (data, fallback = '-', prefix = '', postfix = '') => {",
+          "  return data ? `${prefix}${data}${postfix}` : fallback;",
+          "};",
+          "",
+          "// Usage:",
+          "console.log(dataWithPrefix('123', '-', '(', ')')); // Output: '(123)'",
+          "console.log(dataWithPrefix('', '-', '(', ')')); // Output: '-'",
+          "console.log(dataWithPrefix('Hello', 'N/A', 'Mr. ', '')); // Output: 'Mr. Hello'",
+          "console.log(dataWithPrefix(null, 'N/A', 'Mr. ', '')); // Output: 'N/A'"
+        ],
+        "tags": ["javascript", "data", "utility"],
+        "author": "realvishalrana"
+      }
+    ]
+  },
+  {
+    "language": "javascript",
+    "categoryName": "Object Manipulation",
+    "snippets": [
+      {
+        "title": "Filter Object",
+        "description": "Filters out entries in an object where the value is falsy.",
+        "code": [
+          "export const filterObject = (object = {}) =>",
```

Comment:
This function is useful, but filters out entries if they are 0 , "", ''. I'm not sure if that's what you intended. Maybe you can directly check for null, undefined values if not 🤔 .